src/types/WaterTemperature.ts

```diff
@@ -0,0 +1,3 @@
+export type WaterTemperature = {
+  temperature: { temperature: string } | undefined;
```

Comment:
is this allowed to be undefined? in the `useWaterTemperature` hook it is not marked as optional per `?`. 
and i guess that this type should be `temperature?: string` or what am i missing concerning this nesting of `temperature`? when we return in the hook, it is not nested, right?

Reply:
Wow. I'm gonna do it. 👍